content/docs/en/elements/extras/side-drawer.md

```diff
@@ -0,0 +1,69 @@
+---
+title: SideDrawer
+apiRef: https://docs.telerik.com/devtools/nativescript-ui/Controls/Vue/SideDrawer/overview
+contributors: [sombriks]
+---
+
+`<RadSideDrawer>` is a UI component that provides side drawer navigation for apps.
+
+## Installing the side drawer
+
+```shell
+tns plugin add nativescript-ui-sidedrawer
+```
+
+After that, you need to register it as a vue component:
+
+```javascript
+import RadSideDrawer from "nativescript-ui-sidedrawer/vue";
+Vue.use(RadSideDrawer);
+```
+
+You can do that inside a `.vue` file or inside the `app.js` which bootstraps
```

Comment:
I'm wondering about this part. Lately, the Vue template has been relying entirely on the .vue file approach and almost all app logic goes in there. @rigor789 what's your recommendation?

Also, you don't really need the line break here.

Reply:
Indeed i am not 100% on that one, would be nice to hear @rigor789 . If i take the code which playground generates it should be imported and registered as component inside the .vue file using it.

the best location keeps tricking me because i tried a sub-folder called 'extras' but the generated menu renders the section oddly.

glad for the review, if there is anything else i can do, let me know.